README.md

```diff
@@ -1,13 +1,118 @@
 # terraform-example-foundation
-Example repo showing how the CFT modules can be composed to build a foundation
+This is an example repo showing how the CFT terraform modules can be composed to build a secure GCP foundation. The supplied structure and code is intended to form a starting point for building your own foundation with pragmatic defaults you can customize to meet your own requirements. Currently, the code is built heavily around Cloud Build to allow teams to quickly get started without needing to deploy a CI/CD tool, although it is worth noting the code can easily be executed by your preferred tool.
 
-## Steps
-The repo contains several distinct terraform projects which are dependent on the preceeding steps.
+## Overview
+The repo contains several distinct terraform projects which are layered on top of each other, running in the following order.
 
-1.  [bootstrap](./0-bootstrap/README.md)
-1.  [org](./1-org/README.md)
-1.  [networks](./2-networks/README.md)
-1.  [projects](./3-projects/README.md)
+### 1. [bootstrap](./0-bootstrap/README.md)
```

Comment:
I'm not entirely sure we should keep extensive documentation in the main README as it can definitely fall out of date with the component docs.

Reply:
Yeah agree, my plan was to try and keep things as high level as possible, so people can scan the main README and get an understanding of what it is overall. It is currently a little barren.